README.md

```diff
@@ -15,6 +15,17 @@ see [CHANGELOG.md](CHANGELOG.md)
 * Create [config file](#config-file) `git-conventional-commits init`
 * Adjust config `git-conventional-commits.json` to your needs
 
+#### Global Setup
+* `mkdir -p .git-templates/hooks/`
+* create file `.git-templates/hooks/commit-msg` with following content:
```

Comment:
I think it is easier if we use fully qualified path `~/.git/templates/hooks/commit-msg`

Reply:
Like it too. But i dont know of this path is already used by git itself or some other tools to store something. Maybe brand a folder with .git-conventional-commits? Would be also nice for some global config files.

Reply:
You are right although .git folder is not used by any other tools it would be the git directory if have git versioned home directory :-). The default of `init.templatedir` is `/usr/share/git-core/templates` so I would go for `~/.git-templates` as you suggested.